emails/theme/style.pug

```diff
@@ -17,6 +17,7 @@ style(type="text/css").
 
   a[class="mobile-button"] {font-size: 16px; font-family: Helvetica, Arial, sans-serif; font-weight: normal; color: #ffffff; text-decoration: none; background-color: #6549AA; border-top: 15px solid #6549AA; border-bottom: 15px solid #6549AA; border-left: 25px solid #6549AA; border-right: 25px solid #6549AA; border-radius: 3px; -webkit-border-radius: 3px; -moz-border-radius: 3px; display: inline-block;}
   
+  .mobile-button &:active, .mobile-button &:visited {color: #ffffff;}
```

Comment:
This should probably be:

`.mobile-button:active, .mobile-button:visited { color: #ffffff; }`